minecraft_servers/craftergang/manifest.json

```diff
@@ -1,101 +1,96 @@
-        },
-        "ffa": {
-            "name": "FFA",
-            "color": "#ed0c26",
-            "command": "/gametp ffa",
-            "versions": "1.8<1.20.6"
-        },
-        "bedwars": {
-            "name": "BW",
-            "color": "#e3bcc1",
-            "command": "/gametp bw",
-            "versions": "1.8<1.20.6"
-        },
-        "gungame": {
-            "name": "GunGame",
-            "color": "#FBB300",
-            "command": "/gametp gungame",
-            "versions": "1.8<1.20.6"
-        },
-        "mlgrush": {
-            "name": "MLGRush",
-            "color": "#de0b39",
-            "command": "/gametp mlgrush",
-            "versions": "1.8<1.20.6"
-        },
-        "onehit": {
-            "name": "OneHit",
-            "color": "#f705ab",
-            "command": "/gametp onehit",
-            "versions": "1.8<1.20.6"
-        },
-        "skypvp": {
-            "name": "SkyPvP",
-            "color": "#f7df05",
-            "command": "/gametp spvp",
-            "versions": "1.8<1.20.6"
-        },
-        "survival": {
-            "name": "Survival",
-            "color": "#0d6914",
-            "command": "/gametp survive",
-            "versions": "1.14<1.20.6"
-        }
+    "welt-untergang": {
+      "name": "Untergang",
+      "color": "#0bde19",
+      "command": "/gametp wu",
+      "versions": "1.8<1.21"
     },
-    "discord": {
-        "server_id": 251791624483373060,
-        "rename_to_minecraft_name": false
+    "cbdesert": {
+      "name": "CB Desert",
+      "color": "#0d6914",
+      "command": "/gametp cb1",
+      "versions": "1.8<1.21"
     },
-    "brand": {
-        "primary": "#497cb1",
-        "background": "#245587",
-        "text": "#4646cf"
+    "cbwater": {
+      "name": "CB Water",
+      "color": "#13c7eb",
+      "command": "/gametp cb2",
+      "versions": "1.12<1.21"
     },
-    "location": {
-        "city": "Frankfurt",
-        "country": "Germany",
-        "country_code": "DE"
+    "onehit": {
+      "name": "OneHit",
+      "color": "#f705ab",
+      "command": "/gametp onehit",
+      "versions": "1.8<1.21"
     },
-    "yt_trailer": "Fzf-sy60V7U"
+    "ffa": {
+      "name": "FFA",
+      "color": "#ed0c26",
+      "command": "/gametp ffa",
+      "versions": "1.8<1.21"
+    },
+    "bedwars": {
+      "name": "BW",
+      "color": "#e3bcc1",
+      "command": "/gametp bw",
+      "versions": "1.8<1.21"
+    },
+    "skypvp": {
+      "name": "SkyPvP",
+      "color": "#f7df05",
+      "command": "/gametp spvp",
+      "versions": "1.8<1.21"
+    },
+    "gungame": {
+      "name": "Welt-Untergang",
+      "color": "#0bde19",
+      "command": "/gametp gungame",
+      "versions": "1.8<1.21"
+    },
+    "mlgrush": {
+      "name": "MLGRush",
+      "color": "#de0b39",
+      "command": "/gametp mlgrush",
+      "versions": "1.8<1.21"
+    }
+  },
+  "discord": {
+    "server_id": 251791624483373060,
+    "rename_to_minecraft_name": true

```

Comment:
Reservered for partners only; change this back to `false` if you're not a LabyMod partner